cloudformation-template.template

```diff
@@ -1,17 +1,33 @@
 {
   "AWSTemplateFormatVersion": "2010-09-09",
-  "Parameters": {
-    "s3bucketName": {
+  "Parameters" : {
+    "OriginAPIToken" : {
+      "Type" : "String",
+      "Default" : "12345678",
```

Comment:
do we want to check-in a key into git, I know it is meant to be changed but users may not change it. should it be a mandatory field with no default value instead